src/main/java/org/team1540/robot2024/subsystems/shooter/ShooterPivotIOTalonFX.java

```diff
@@ -97,7 +99,7 @@ public void updateInputs(ShooterPivotIOInputs inputs) {
         inputs.isAtForwardLimit = forwardLimit.getValue() == ForwardLimitValue.ClosedToGround;
         inputs.isAtReverseLimit = reverseLimit.getValue() == ReverseLimitValue.ClosedToGround;
         inputs.position = Rotation2d.fromRotations(position.getValueAsDouble());
-        inputs.absolutePosition = Rotation2d.fromRotations(absolutePosition.getValueAsDouble() / CANCODER_TO_PIVOT * CHAIN_FACTOR);
+        inputs.absolutePosition = Rotation2d.fromRotations(absolutePosition.getValueAsDouble() / CANCODER_TO_PIVOT * CHAIN_FACTOR).plus(Rotation2d.fromRadians(0.03865 - 0.02649405));
```

Comment:
why are these magic numbers being subtracted from one another?

Reply:
Probably to fix the new cancoder. We'd probably want to retune this at the lab either way.